src/pages/home/home.ts

```diff
@@ -39,17 +39,21 @@ export class HomePage {
   }
 
   reset() {
-    this.navCtrl.setRoot(this.navCtrl.getActive().component);
+    this.slides.slideTo(0, 500);
+    this.cardSupply = this.prov.allCards();
   }
```

Comment:
the problem with this soluton is that if a card hase been left "flipped", it is still flipped when you go back to 0 and go past them again. I need a way to reset all the "flipped" attributes of the card deck to reset() it...